docker-runner.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-scriptdir="$(cd "${0%/*}"; pwd)"
-${scriptdir}/tools/docker/run.sh "$@"
+scriptdir="$(cd "${0%/*}" || exit 1; pwd)"
```

Comment:
As mentioned elsewhere: I prefer
```suggestion
scriptdir="$(cd "${0%/*}" && pwd)"
```
but that's really personal preference.